src/pages/Chat/ChatConversation.tsx

```diff
@@ -0,0 +1,100 @@
+import { Container, Provider, StateUpdater } from 'constate'
+import { always, ifElse } from 'ramda'
+import * as React from 'react'
+import { ChatMessage } from '../../components/hedvig/chat'
+import { TimedMount } from '../../components/utils/TimedMount'
+import { Greet } from './steps/Greet'
+import { NameAgeInput } from './steps/NameAgeInput'
+import { ChatWrapper } from './styles'
+
+export type MessageRenderer = (next: () => void) => React.ReactNode
+
+export interface Message {
+  render: MessageRenderer
+  delay?: number
+  key?: string | number
+}
+
+export const chatMessages = (): Message[] => [
+  {
```

Comment:
🤷‍♀️ i guess smaken är som baken, i think this gets easier to read, especially considering we'll want to do things like "come back later" or branching the flow

Reply:
I guess this might be for prototyping purposes but I find it slightly weird that `chatMessages()` and `ChatConversation` is colocated but exported separately then brought together again in a separate file 🤔

Reply:
hm, fair enough

Reply:
I'm a bit skeptical to this structure as well. Could it be solved better with JSX?

Reply:
sure, I'll look into it 👍